config/cluster.config

```diff
@@ -10,6 +10,7 @@ process {
 executor {
   $lsf {
     queueSize = 10000
+    submitRateLimit = '1sec'
```

Comment:
I added this to ensure e! and ENA worked this time around - we should probably review it for the next release